docs/weblog/README.md

```diff
@@ -229,6 +229,9 @@ A GET request using a cookie with name `table` and any value. The value must be
 
 A GET request using a cookie with name `user` and any value. The name must be used in the vulnerability.
 
+#### POST /iast/source/multipart/test
+a POST request 
```

Comment:
Could be a little bit better ;)

Reply:
The comment got truncated by mistake. Fixed